compiler/src/main/java/dagger/internal/codegen/BindingGraph.java

```diff
@@ -55,6 +57,8 @@
 import com.google.common.collect.Sets;
 import com.google.common.collect.TreeTraverser;
 import com.google.common.util.concurrent.ListenableFuture;
+import com.sun.tools.javac.code.Symbol;
```

Comment:
There's also a big warning at the top of the javadoc: 
**This is NOT part of any supported API. If you write code that depends on this, you do so at your own risk. This code and its internal interfaces are subject to change or deletion without notice.**